### DISCUSSION TITLE
Expose dragPan config option from Mapbox

### DISCUSSION THREAD

**Discussion by franciscohanna92** [rnmapbox/maps#1347] [Q&A]
I have an app with two tabs for navigation where swiping to change tabs is enabled. In of the tabs I have a Mapbox instace. 

I want to disabled the drag pan gesture (https://docs.mapbox.com/mapbox-gl-js/api/handlers/#dragpanhandler), and allow the user to use two fingers to pan the map. I don't think it's possible on this library at the moment. Or is it? I couldn't find it. 

Any ideas for a workaround? Trying to swipe tabs and that instead the map moves kind of breaks the UX.

**Comment by ferdicus** [comment]
hey there, 
this repo wraps around the native mobile SDKs, and feature parity with the JS lib isn't guaranteed. 

If you can find this feature exposed on the Android and iOS SDKs for mapbox let us know and we can work something out. 
Better yet, PRs are always welcome :)
